src/controller/payments.controller.ts

```diff
@@ -16,25 +16,26 @@ import {
   UserAlreadyExistsError,
   CustomerNotFoundError,
   InvalidTaxIdError,
-} from './services/PaymentService';
-import fastifyJwt from '@fastify/jwt';
-import { User, UserSubscription, UserType } from './core/users/User';
-import CacheService from './services/CacheService';
+} from '../services/payment.service';
+import { User, UserSubscription, UserType } from '../core/users/User';
+import CacheService from '../services/cache.service';
 import Stripe from 'stripe';
 import {
   InvalidLicenseCodeError,
   LicenseCodeAlreadyAppliedError,
   LicenseCodesService,
-} from './services/LicenseCodesService';
-import { Coupon } from './core/coupons/Coupon';
+} from '../services/licenseCodes.service';
+import { Coupon } from '../core/coupons/Coupon';
+import { assertUser } from '../utils/assertUser';
+import fastifyJwt from '@fastify/jwt';
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const rateLimit = require('fastify-rate-limit');
 
 type AllowedMethods = 'GET' | 'POST';
 
 export const allowedCurrency = ['eur', 'usd'];
 
-const allowedRoutes: {
+const ALLOWED_DOMAINS: {
```

Comment:
These are allowedPaths or routes, these are not domains

Reply:
Name updated.